app/gui/src/project-view/components/visualizations/TableVisualization.vue

```diff
@@ -81,12 +81,7 @@ interface UnknownTable {
   get_child_node_action: string
   get_child_node_link_name: string
   link_value_type: string
-  data_quality_pairs?: DataQualityPairs
-}
-
-interface DataQualityPairs {
-  number_of_nothing: number[]
-  number_of_whitespace: number[]
+  data_quality_pairs?: Record<string, number[]>[]
```

Comment:
Why an array of single-key objects? If the libs can provide them as a single `Record`, they'd be easier to work with.

Reply:
+1, wouldn't the interface between libs and visualization be more robust if we defined some fields, e.g. `name` and `percentage`? Perhaps then in the future, it would be easier to introduce other 'kinds' of metrics (that rely on something else than percentage) without breaking the interface (as much).